pkgs/applications/science/math/mxnet/default.nix

```diff
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake perl ];
 
-  buildInputs = [ opencv3 gtest blas ]
+  buildInputs = [ opencv3 gtest openblas ]
```

Comment:
```suggestion
  buildInputs = [ opencv3 gtest blas blas.provider ]
```

Reply:
ah, wasn't aware of the provider usage

Reply:
is there any need for the actual `blas` package/attr set? Seems to build fine with only the provider

Reply:
oh, linking maybe

Reply:
Build just failed for me; looks like we need to do the same with `lapack` to make it visible to the linker.

Reply:
> is there any need for the actual blas package/attr set? Seems to build fine with only the provider

Not actually sure ... in principle it might be sufficient to just pass in `blas.provider`, but it seems reasonable to expose the meta-package as well, as that has the actual public API shared objects defined.